src/Microsoft.ML.Data/Transforms/TermEstimator.cs

```diff
@@ -11,7 +11,7 @@
 
 namespace Microsoft.ML.Runtime.Data
 {
-    public sealed class TermEstimator : IEstimator<TermTransform>
+    public sealed class ToKeyEstimator : IEstimator<TermTransform>
```

Comment:
>ToKey [](start = 24, length = 5)

There are three estimators that work around keys: `ToKey`, `ToValue` and `ToVector`. 
I suggest calling them `ValueToKeyConvertingEstimator`, `KeyToValueConvertingEstimator` and `KeyToVectorConvertingEstimator`.

Reply:
I'm not sure that I like the idea of "conversion" since the name "conversion" does not suggest to me what this is doing, which is learning what amounts to a one-to-one mapping from a set of values, to a set of keys. So maybe, `ValueToKeyMapping` instead of `KeyToValueConverting`.

Reply:
Taking Tom's suggestion and using Mapping instead of converting 

---
In reply to: [227051421](https://github.com/dotnet/machinelearning/pull/1328#discussion_r227051421) [](ancestors = 227051421)